src/v/cluster/archival/ntp_archiver_service.cc

```diff
@@ -1796,12 +1796,7 @@ ss::future<ntp_archiver_upload_result> ntp_archiver::upload_segment(
     // the read path will create the index on the fly while downloading the
     // segment, so it is okay to ignore the index upload failure, we still
     // want to advance the offsets because the segment did get uploaded.
-
-    ssx::spawn_with_gate(
-      _gate,
-      [this, path = std::move(index_path), index = std::move(index)]() mutable {
-          return upload_index(std::move(path), std::move(index));
-      });
```

Comment:
nit: in the commit message, mind linking to the PR/commit that backgrounded this?

Reply:
done